src/r/DESCRIPTION

```diff
@@ -7,10 +7,10 @@ Authors@R: c(person("Hongyuan", "Jia", email = "hongyuan.jia@bears-berkeley.sg",
              person("David", "Meyer", role = "aut", comment = c(ORCID = "0000-0002-7071-7547")),
              person("Didier", "Thevenard", role = "aut", comment = c(ORCID = "0000-0002-0749-6841")))
 Description:
-    R implementation of 'PsychroLib'
+    Implementation of 'PsychroLib'
     <https://github.com/psychrometrics/psychrolib> to calculate psychrometric
     properties of moist and dry air in both metric (SI) and imperial
-    (IP) systems of units.
+    (IP) systems of units. References: Meyer, D. and Thevenard, D (2019) <doi.org/10.21105/joss.01137>.
 Depends:
     R (>= 3.0.0)
 Imports:
```

Comment:
I know we have been here before but given what is done at https://github.com/rstudio/rmarkdown/blob/master/DESCRIPTION#L108, I would remove `+ file LICENSE`

Reply:
So you would like to change LICENSE to GPL-3?

Reply:
No just `MIT` instead of `MIT + file LICENSE`

Reply:
No, CRAN does not allow a package licensed MIT without file license.

See here: https://kbroman.org/pkg_primer/pages/licenses.html

Reply:
My bad -- sorry! Fine as it is.

Reply:
No worries. I will try to resubmit again now.

Reply:
Sounds good and good luck! 🤞